### ISSUE TITLE
Buttons for "Skip" or "Done" actions

### ISSUE THREAD

**Issue by mikaelkundert** [roldanjr/pomatez#237]
I feel little bit frustrated to select "Skip" or "Done" task from the action menu. Especially when skipping multiple tasks at the same time to seek for something that is doable within a task list.

Every time you have to skip, you need two clicks in the timer view.

Having shortcut buttons to click directly above/below/corner of the task would make it way faster and intuitive to use.

Using shortcut keys would perhaps work also.

![image](https://user-images.githubusercontent.com/93661/161779316-66f461da-78ba-4d93-9dbe-41105c0d0e1f.png)

**Comment by sekwah41** [comment]
Agreed, I also want to take a look at adding a quick button for toggling done to the list 
![image](https://github.com/zidoro/pomatez/assets/3209834/c9648d23-9e41-4aea-a1af-1c5bda257f7f)
as there is already delete and edit.

**Comment by mikaelkundert** [comment]
I started to use again after changing jobs. First improvement idea came into my mind again, and apparently I did wrote an issue about this. So, I'm +1 my own issue after forgetting it 😆 

It looks that the source code is somewhere at https://github.com/zidoro/pomatez/blob/d802264f6e01f1b6ba4d4477d09a1e746d4b7f97/app/renderer/src/routes/Timer/PriorityCard.tsx#L80C9-L80C32
